04-spatial-operations.Rmd

```diff
@@ -26,17 +26,22 @@ library(spData)
 Spatial operations are a vital part of geocomputation.
 This chapter shows how spatial objects can be modified in a multitude of ways based on their location and shape.
 The content builds on the previous chapter because many spatial operations have a non-spatial (attribute) equivalent.
-Spatial operations on *vector* objects include spatial subsetting (covered in section \@ref(spatial-subsetting)) and spatial joining (section \@ref(spatial-joining)).
-This content builds on section \@ref(vector-attribute-manipulation) in the previous chapter.
-Spatial operations on *rasters* include merging and subsetting, covered in section \@ref(spatial-ras).
-
-The chapter also introduces new concepts that are unique to spatial data.
-A variety of *topological relations* can be used to subset/join vector geometries (by default **sf** uses the catch-all *intersects* but other relations such as *within* can be very useful), a topic that is explored in section \@ref(topological-relations).
-Spatial operations on raster datasets involve *map algebra* (covered in sections \@ref(map-algebra) to \@ref(global-operations-and-distances)) and raster merging (covered in sections \@ref(merging-rasters)).
-
+This is especially true for *vector* operations.
+For instance, we will extend the basics on vector attribute manipulation (covered in section \@ref(vector-attribute-manipulation)) with its spatial counterpart, namely spatial subsetting (covered in section \@ref(spatial-subsetting)) and spatial joining (section \@ref(spatial-joining)).
+Imagine you want to add an attribute of a road (such as the road number or the speed limit) to a municipality polygon.
+Equally, you might want to select all roads of a specific municipality.
+Were we in possession of an ID this would simply repeat the stuff introduced in the previous chapter. 
+But if we are missing an ID, then we can still solve these questions using the geographical location of the objects in question.
+But then we might face further questions such as: Should the road completely fall inside the polygon or is it enough if the road simply crosses the municipality or is it even okay if it is within a certain distance of it.
```

Comment:
Again overly informal language: there's a whole article urging teachers not to use words like 'simply', especially for programming: https://medium.freecodecamp.org/one-does-not-simply-learn-to-code-f25bacdc5b62

Reworded - see what you think and let me know if there's any other issues. I agree with the direction of your rewording though, great to have a concrete example of roads (makes me think of more content in Part II or IV).

Reply:
I like your rewrite, and yes, this sentence, too, was too informal, sorry. And we don't have to use "simply". But here simply didn't refer to coding but if a road crosses a polygon. Nevertheless, we can omit the word. 
Regarding the "simply" post, I understand what the author wants to say. On the other hand, I would also point out that coding is not rocket science, and of course, everybody can learn it. But like with any other language or skill it takes time to become proficient.
Still, I will try to omit "simply" in the future.

Reply:
Fair point - simply has its place and agree that it was referring to a road not code.